master/usage/troubleshooting/logging.md

```diff
@@ -34,57 +34,52 @@ See the following sub-sections for details on configuring the log level for
-command on a specific host to change the log level for that specific BGP node.
+Use the `calicoctl config loglevel` command on any host to change the
+log level across all Calico nodes, _or_ use the same command with the `--node`
+option to run the command for that specific node.  This command affect the 
+logging level for both Bird/Bird6 and Felix.
 
-Valid log levels are:  none, debug and info.  e.g.
+Valid log levels are:  none, debug, info, warning, error, critical
 
-        calicoctl config bgp loglevel none
-        calicoctl config node bgp loglevel debug
+        calicoctl config loglevel error
 
 ### Felix
 Felix is the primary Calico agent that runs on each machine that hosts
 endpoints.  Felix is responsible for the programming of iptables rules on the
-host.
+host.  The logs are output in the `felix` sub-directory of the calico/node 
+logging directory.
 
-Directory | Default level
---- | ---
-`felix` | `info`
-
-Use the `calicoctl config felix loglevel` command on any host to change the
-log level on all Felix instances.
+Use the `calicoctl config loglevel` command on any host to change the
+log level across all Calico nodes, _or_ use the same command with the `--node`
+option to run the command for that specific node.  This command affect the 
+logging level for both Bird/Bird6 and Felix.
 
 Valid log levels are:  none, debug, info, warning, error, critical
 
-        calicoctl config felix loglevel error
+        calicoctl config loglevel error --node=Calico-Node-1
 
 ### confd
 The confd agent generates configuration files for Felix and Bird using
-configuration data present in the etcd datastore.
-
-Directory | Default level
---- | ---
-`confd` | `DEBUG`
+configuration data present in the etcd datastore.  The logs are output in the
+`confd` sub-directory of the calico/node logging directory.
 
-To change the log level, edit the node_filesystem/etc/service/confd/run and
-rebuild the calico-node image.
+By default, the confd logging level is "debug" and cannot be changed without
+editing configuration within the node image.  To change the log level, checkout
+the calico-containers codebase, edit the node_filesystem/etc/service/confd/run
+and rebuild the calico-node image.
 
 For more information on the allowed levels, see the
 [documentation](https://github.com/kelseyhightower/confd/blob/master/docs/configuration-guide.md)
 
 
 ## Docker network and IPAM driver
-When running Calico as a Docker network plugin, the Calico driver is run inside the calico/node container.
-
-The logs may be viewed running the standard `docker logs` command for this
-container.  e.g.
+When running Calico as a Docker network plugin, the Calico network driver is run
+inside the calico/node container.  The logs are output in the `libnetwork` sub-directory
+of the calico/node logging directory.
 
-    docker logs calico-node
+The logging level of the libnetwork plugin is fixed.
```

Comment:
I'd say just delete this line, then.  Alternatively, I think 'cannot be changed' is better than 'is fixed'.  The latter sounds like it might be an incomplete sentence.